core/src/main/resources/hudson/model/queue/_api.jelly

```diff
@@ -14,4 +14,4 @@
     </ul>
   </p>
 
-</j:jelly>
```

Comment:
I made the update directly from github UI, it added the trailing line jump automatically, let me know if it's an issue I'll checkout the file from my IDE.